tools/build_pak_files.py

```diff
@@ -15,26 +15,20 @@
 if __name__ == '__main__':
 
     parser = argparse.ArgumentParser()
-    parser.add_argument("--unreal_engine_dir", required=True)
-    parser.add_argument("--perforce_content_dir", required=True)
+    parser.add_argument("--unreal_engine_dir", required=True)    
     parser.add_argument("--output_dir", required=True)
     parser.add_argument("--version_tag", required=True)
+    parser.add_argument("--skip_create_symlinks", action="store_true")
+    parser.add_argument("--perforce_content_dir")
     parser.add_argument("--scene_ids")
     args = parser.parse_args()
-    
-    assert os.path.exists(args.unreal_engine_dir)
```

Comment:
let's keep `assert os.path.exists(args.unreal_engine_dir)` because we generally validate input arguments as early as possible

Reply:
Done